client/my-sites/marketing/connections/service.jsx

```diff
@@ -294,9 +303,10 @@ export class SharingService extends Component {
 	 * Fetch connections
 	 *
 	 * @param {Object} connection Connection to update.
+	 * @return {Function} Action thunk
 	 */
 	fetchConnection = connection => {
-		this.props.fetchConnection( this.props.siteId, connection.ID );
+		return this.props.fetchConnection( this.props.siteId, connection.ID );
```

Comment:
To makes sure we can use the `Promise`

Reply:
Could use arrow function without brackets here

```
fetchConnection = connection => this.props.fetchConnection( this.props.siteId, connection.ID );
```